build.gradle

```diff
@@ -114,6 +114,8 @@ run {
     useCluster testClusters.integTest
 }
 
+dependencyLicenses.enabled = false
```

Comment:
oh, i just saw this: this is a leftover from when i wrongly added the `org.apache.hc.core5`dependency and should've been removed again (since now we don't have any dependency here) :|

i can provide a PR for that next week (or someone else can briefly remove this). sorry about that!